doc/source/whatsnew/v0.17.0.txt

```diff
@@ -80,6 +80,15 @@ Other enhancements
 
 - ``regex`` argument to ``DataFrame.filter`` now handles numeric column names instead of raising ``ValueError`` (:issue:`10384`).
 
+- Reindex now has a ``tolerance`` argument that allows for finer control of :ref:`basics.reindex_fill`:
+
+  .. ipython:: python
+
```

Comment:
would add a small section in ``missing.rst`` (and link to from here), that shows off ``tolerance/nearest``